src/md-collection-loader/index.js

```diff
@@ -100,6 +96,10 @@ module.exports = function(input) {
     ...metadata,
   }
 
+  if (query.description) {
+    mdObject = description(mdObject, query.description)
+  }
+
```

Comment:
Don't think let is necessary, this make the code not clear. Btw I am not sure we should rely and some html output. This makes things complicated.